### PR TITLE
Add lines connecting body and arms

### PR DESCRIPTION
**Pull request by YounseoRyu**
This example of painting lines that connect joints will be useful for developers.

### CODE REVIEW

File: example/lib/VisionDetectorViews/painters/pose_painter.dart [hunk 1]
```diff
@@ -29,6 +29,70 @@ class PosePainter extends CustomPainter {
...
+        ..strokeWidth = 6
+        ..color = Colors.yellow;
+
+      final rightPaint = Paint()
+        ..style = PaintingStyle.stroke
+        ..strokeWidth = 6
+        ..color = Colors.blueAccent;
+
+      PoseLandmark leftShoulder =
+          pose.landmarks[PoseLandmarkType.leftShoulder]!;
```
Comment (fbernaly) [inline example/lib/VisionDetectorViews/painters/pose_painter.dart:44]: What happend when the landmark is null? could you cover that case as well?

File: example/lib/VisionDetectorViews/painters/pose_painter.dart [hunk 2]
```diff
@@ -29,6 +29,70 @@ class PosePainter extends CustomPainter {
             1,
             paint);
       });
+
+      final leftPaint = Paint()
+        ..style = PaintingStyle.stroke
+        ..strokeWidth = 6
```
Comment (fbernaly) [inline example/lib/VisionDetectorViews/painters/pose_painter.dart:35]: make the strokeWidth no greater than the strokeWidth of the paint used for the dots

Comment (YounseoRyu) [reply example/lib/VisionDetectorViews/painters/pose_painter.dart:35]: Okay, I will make the changes in accordance with your reviews. Thank you!

File: example/lib/VisionDetectorViews/painters/pose_painter.dart (hunk 1 above)
Comment (YounseoRyu) [reply example/lib/VisionDetectorViews/painters/pose_painter.dart:44]: My understanding is: if there is a pose detected, there is no case where any landmark is null.
Even when some joints are not detected (i.e. legs are not detected as a user is showing the upper body only), the google ml kit still predicts the coordinates of the missing joints.

Comment (fbernaly) [reply example/lib/VisionDetectorViews/painters/pose_painter.dart]: ok, let me test those changes

**Review by fbernaly — APPROVED** [review]

**Comment by fbernaly** [comment]
@YounseoRyu: Thanks for your contribution.

**Pull request merged**